mysql/docker-entrypoint-initdb.d/02_data.sql

```diff
@@ -356,4 +365,88 @@ CREATE PROCEDURE INSERTROWS(NUM INT) BEGIN
 
 DELIMITER ;
 
-CALL InsertRows(250);
\ No newline at end of file
+CALL InsertRows(250);
+
+CREATE TABLE `Labels` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `label` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `Unique_label` (`label`) USING BTREE
+)
+;

```

Comment:
there is a file for tables :)

Reply:
moved 
[move create table to 01_tables.sql](https://github.com/Bot-detector/public-api/pull/25/commits/75f4cbf2abc95b58bf0ac9641be15083a3cfdb1c)